apps/healthlocker/web/views/message_view.ex

```diff
@@ -14,20 +14,44 @@ defmodule Healthlocker.MessageView do
   end
 
   @base_classes "w-80 br2 mb2 pa1 pa3-ns"
-  @sender_classes "hl-bg-light-yellow fr"
-  @receiver_classes "hl-bg-light-aqua fl"
+  @align_right "fr"
+  @align_left "fl"
+  @sender_classes "hl-bg-light-yellow"
+  @receiver_classes "hl-bg-light-aqua"
+  @teacher_classes "hl-bg-lilac"
 
   def classes(_message, nil) do
     @base_classes
   end
 
   def classes(message, current_user_id) do
-    communicator = if message.user.id == current_user_id do
-      @sender_classes
+    alignement = get_alignement(message.user, current_user_id)
+    colour = get_message_colour(message.user, current_user_id)
+
+    @base_classes <> " " <> alignement <> " " <> colour
+  end
+
+  defp message_from_current_user?(message_user, current_user_id) do
+    message_user.id == current_user_id
+  end
+
+  defp get_alignement(message_user, current_user_id) do
+    if message_from_current_user?(message_user, current_user_id) do
+      @align_right
     else
-      @receiver_classes
+      @align_left
     end
+  end
 
-    @base_classes <> " " <> communicator
+  defp get_message_colour(message_user, current_user_id) do
```

Comment:
I do not think this `if` is working as expected...

![image](https://user-images.githubusercontent.com/15571853/35675431-e0bcb73e-073f-11e8-8b8d-198dc884646c.png)

It might be a little bit more readable and easier to fix the logic if you use a `cond` statement instead of nested `ifs`

Reply:
Thanks for spotting this @RobStallion I think it's linked to the logic on the Phoenix channel, I'll see if I can enhance this part of the code 👍

Reply:
@RobStallion I've updated the function and the colour for teachers should be correct now